### DISCUSSION TITLE
is there a way to make the scrollmarkers disabled? or atleast make it toggleable?

### DISCUSSION THREAD

**Discussion by AdrianeYves** [alefragnani/vscode-bookmarks#734] [Q&A] [answered]
i already have tons of scroll markers related to git and i dont want the bookmarkers scroll marks add more to it.
![image_2024-07-26_125437452](https://github.com/user-attachments/assets/3b83621c-cf8b-47a9-8110-e479b184ebe2)

**Comment by alefragnani** [comment, marked as answer]
Hi @AdrianeYves ,

No, not today. There is no setting to choose its visibility, only its color. But it could be added.

Thanks for your suggestion

**Reply by AdrianeYves** [reply]
Thanks!

**Comment by cas--** [comment]
For a long time I have been wondering about this too since the colour and size of bookmark indicator in the ruler is more prominent and conflated with other markers such as git. Reading through all the configuration option here I finally found the overview ruler colour customisation which also controls transparency.

So to hide bookmarks in the ruler you can set transparency to zero:

```json
  "workbench.colorCustomizations": {
    "bookmarks.overviewRuler": "#00000000"
  },
```

Perhaps the overview rule colour can be exposed in the UI settings like the gutter colour?

**Reply by alefragnani** [reply]
Hi @cas-- ,

Yes, you can customize the color, and transparency, of the overview ruller, and that's the objective of this setting. The reason why it is a `colorCustomization` instead of a regular setting, is because doing so, themes can provide customization as well, to follow its identity. 

The gutter icon is a _regular setting_ because the icon is created on the fly, and it needs to use hex values, but VS Code does not provide yet an API to extract the hex value from a color.

Hope this helps
